edx_lint/files/.editorconfig

```diff
@@ -0,0 +1,25 @@
+# This is a file to standardize editor settings: http://EditorConfig.org
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
+charset = utf-8
```

Comment:
Yes, I'd like to. Although, per your in-person suggestion, maybe I'll just pull the editorconfig out into a separate repo.